<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>541</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="bowlFrame" name="bowlFrame_4">
       <property name="ourFrameNumber" stdset="0">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="bowlFrame" name="bowlFrame_8">
       <property name="ourFrameNumber" stdset="0">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="bowlFrame" name="bowlFrame_7">
       <property name="ourFrameNumber" stdset="0">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="bowlFrame" name="bowlFrame_5">
       <property name="ourFrameNumber" stdset="0">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="bowlFrame" name="bowlFrame_9">
       <property name="ourFrameNumber" stdset="0">
        <number>9</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="bowlFrame" name="bowlFrame_6">
       <property name="ourFrameNumber" stdset="0">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="bowlFrame" name="bowlFrame_3">
       <property name="ourFrameNumber" stdset="0">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="bowlFrame" name="bowlFrame_2">
       <property name="ourFrameNumber" stdset="0">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="bowlFrame" name="bowlFrame_10">
       <property name="ourFrameNumber" stdset="0">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="bowlFrame" name="bowlFrame_1"/>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>301</width>
      <height>551</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ball Roll #1</string>
     </property>
     <widget class="bowlPinMap" name="bowlPinMap_ball1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>291</width>
        <height>281</height>
       </rect>
      </property>
      <property name="ourTurnNumber" stdset="0">
       <number>1</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Ball Roll #2</string>
     </property>
     <widget class="bowlPinMap" name="bowlPinMap_ball2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>291</width>
        <height>281</height>
       </rect>
      </property>
      <property name="ourTurnNumber" stdset="0">
       <number>2</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_FrameNumber">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Frame #</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_FrameNumber_Val">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>#</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>splitter</zorder>
   <zorder>splitter_2</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>bowlFrame</class>
   <extends>QWidget</extends>
   <header>bowlframe.h</header>
  </customwidget>
  <customwidget>
   <class>bowlPinMap</class>
   <extends>QWidget</extends>
   <header>bowlpinmap.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>bowlFrame_1</sender>
   <signal>mouseClick_OurFrameNum(int)</signal>
   <receiver>label_FrameNumber_Val</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bowlFrame_2</sender>
   <signal>mouseClick_OurFrameNum(int)</signal>
   <receiver>label_FrameNumber_Val</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bowlFrame_3</sender>
   <signal>mouseClick_OurFrameNum(int)</signal>
   <receiver>label_FrameNumber_Val</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bowlFrame_4</sender>
   <signal>mouseClick_OurFrameNum(int)</signal>
   <receiver>label_FrameNumber_Val</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bowlFrame_5</sender>
   <signal>mouseClick_OurFrameNum(int)</signal>
   <receiver>label_FrameNumber_Val</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bowlFrame_6</sender>
   <signal>mouseClick_OurFrameNum(int)</signal>
   <receiver>label_FrameNumber_Val</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bowlFrame_7</sender>
   <signal>mouseClick_OurFrameNum(int)</signal>
   <receiver>label_FrameNumber_Val</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bowlFrame_8</sender>
   <signal>mouseClick_OurFrameNum(int)</signal>
   <receiver>label_FrameNumber_Val</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bowlFrame_9</sender>
   <signal>mouseClick_OurFrameNum(int)</signal>
   <receiver>label_FrameNumber_Val</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bowlFrame_10</sender>
   <signal>mouseClick_OurFrameNum(int)</signal>
   <receiver>label_FrameNumber_Val</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
